### PR TITLE
add "iz", close #177

### LINKED ISSUE #177: How about adding iz ? It has revealed another reg today
I've already been in contact about regressions in [IZ](https://github.com/BBasile/iz) with M.Nowak. Today again another reg revealed in this lib: https://issues.dlang.org/show_bug.cgi?id=18469

I test it on [SemaphoreCI against the nighlty](https://semaphoreci.com/bbasile/iz) since beginning of 2018 but for some reason this REG wasn't detected.

### PR DESCRIPTION
**Pull request by BBasile**
- Motivated by 2 or 3 regs per year due to the diversity of style present in this lib.
- runs in 25 secs on SemaphoreCI, might be slightly slower here.

### CODE REVIEW

**Comment by BBasile** [comment]
Well the failure is actually caused by the regression that motivated this addition, so everything is fine.

**Comment by BBasile** [comment]
All right for me. Everything is done correctly. Up to to you to decide.

File: vars/runPipeline.groovy [hunk 1]
```diff
@@ -462,6 +466,7 @@ def call() { timeout(time: 1, unit: 'HOURS') {
         "libmir/mir-algorithm", // 17s
         "CyberShadow/ae", // 22s
         "dlang-community/DCD", // 23s
+        "BBasile/iz",
```
Comment (wilzbach) [inline vars/runPipeline.groovy]: Could you please set this to `12s` and put it at the according position?

Or enable PR push to your PRs ;-)

```
 ! [remote rejected] HEAD -> add-iz (permission denied)
error: failed to push some refs to 'git@github.com:BBasile/ci'
```

File: vars/runPipeline.groovy [hunk 2]
```diff
@@ -456,12 +460,13 @@ def call() { timeout(time: 1, unit: 'HOURS') {
...
         // sorted by test time fast to slow (to minimize pending queue length)
         "dlang-community/dfmt", // 11s
+        "BBasile/iz", //12s
         "dlang-community/libdparse", // 13s
         "libmir/mir-random", // 13s
         "dlang-community/D-YAML", // 15s
         "libmir/mir-algorithm", // 17s
         "CyberShadow/ae", // 22s
-        "dlang-community/DCD", // 23s
+        "dlang-community/DCD", // 23s
```
Comment (wilzbach) [inline vars/runPipeline.groovy]: Unrelated trailing whitespace.

File: vars/runPipeline.groovy [hunk 3]
```diff
@@ -456,12 +460,13 @@ def call() { timeout(time: 1, unit: 'HOURS') {
         "libmir/mir",
         // sorted by test time fast to slow (to minimize pending queue length)
         "dlang-community/dfmt", // 11s
+        "BBasile/iz", //12s
```
Comment (wilzbach) [inline vars/runPipeline.groovy]: Also I had to be a style facist, but all the other tests use `// Xs` ...
(I would usually fix such nits myself)

Comment (BBasile) [reply vars/runPipeline.groovy]: if so you would have made your own check in dscanner for the style.

**Comment by BBasile** [comment]
should be good now.
